libmscore/score.cpp

```diff
@@ -3950,6 +3982,44 @@ int Score::nmeasures() const
       return n;
       }
 
+//---------------------------------------------------------
```

Comment:
The fun with this was having it double as a function for getting the score's first trailing measure, and also having it serve as setting the appropriate rest for a specific staff if there was a valid selection from the caller (Score::move()). I used a default nullptr rather than function overloading, which required a double pointer rather than a reference since references can't be null